packages/terra-list/CHANGELOG.md

```diff
@@ -3,19 +3,22 @@
 ## Unreleased
 
 * Changed
-  * Added a `ariaSelectionStyle` prop that provides accessibility support for single and multi select lists
   * Fix section and subsection header colors in the lowlight theme.
 
-## 4.52.0 - (December 7, 2022)
-
+## 4.53.0 - (January 3, 2023)
 * Changed
-  * Sections and subsections updated to add programmatic context to communicate hierarchal relationships
+  * Added a `ariaSelectionStyle` prop that provides accessibility support for single and multi select lists
```

Comment:
Is this correct or is there some weird merge collapse hitting here?

Reply:
This is correct. The reason this looks weird is because the below section with the `ariaDescribedBy`, `ariaDescription` and `ariaDetails` props was mistakenly under the 4.52.0 section, probably while fixing merge conflicts before, and that's fixed here.

Reply:
Never mind, misunderstood your meaning. Updated here [1baa72c](https://github.com/cerner/terra-core/pull/3709/commits/1baa72c3642716667b30f6451d84ef4be1b21ab0)